### ISSUE TITLE
[Payment Due Mon 19 April] Accessibility report for Expensify.cash.

### ISSUE THREAD

**Issue by gauravahir28** [Expensify/Expensify.cash#2405]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/master/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Expected Result:
The [Expensify.cash](https://expensify.cash/) site should be accessible to all the users regardless of disability.
To make sure that the [Expensify.cash](https://expensify.cash/) site is accessible to everyone, it needs to be compliant to WCAG 2.1 
## Actual Result:
The [Expensify.cash](https://expensify.cash/) site doesn't  seem to be compliant with  WCAG 2.1.
Therefore, It is not Accessible to Users with vision impairment, Motor impairment, Color blind, Low vision etc.
## Action Performed:
We performed Accessibility checks on following workflows:

- Creating an account
- Understand which conversations contain unread messages, and select a conversation
- Reading messages in a conversation thread
- Sending a message to a user

For step 1, We have Created a detailed report with screenshots and selector/XPath of the elements on which the issues have occured.
For steps 2 to 4, We have created a detailed document to make the chat application accessible.
Both report and document have been attached with this issue. 
## Workaround:
Unless the [Expensify.cash](https://expensify.cash/) site isn't made accessible, it will not be usable for Users with vision impairment, Motor impairment, Color blind, Low vision etc.
## Platform:
Where is this issue occurring?

 Web: Yes.
 iOS: Not Tested as communicated over Upwork
 Android: Not Tested as communicated over Upwork
 Desktop App: Not Tested as communicated over Upwork
 Mobile Web: Not Tested as communicated over Upwork

*Version Number:*
*Logs:* https://stackoverflow.com/c/expensify/questions/4856
*Notes/Photos/Videos:* Any additional supporting documentation
*Expensify/Expensify Issue URL:*
Upwork job URL: [](https://www.upwork.com/jobs/React-Native-WCAG-Accessibility-Testing-and-Audit-Expensify_~012eef4487da592a14)
Accessibility report:  [Tekvision_Expensify.cash_Accessibility_Testing_2021_04_14.xlsx](https://github.com/Expensify/Expensify.cash/files/6317212/Tekvision_Expensify.cash_Accessibility_Testing_2021_04_14.xlsx)
Screenshots: [Expensify Images.zip](https://github.com/Expensify/Expensify.cash/files/6317213/Expensify.Images.zip)
Document to make chat application accessible: [Tekvision-Expensify_Chat-Application-Accessibility_2021-04-15.pdf](https://github.com/Expensify/Expensify.cash/files/6317214/Tekvision-Expensify_Chat-Application-Accessibility_2021-04-15.pdf)

**Comment by laurenreidexpensify** [comment]
@kadiealexander Have you already issued payment for this, I couldn't work it out in Upwork? If so, can we close? Thanks!

**Comment by kadiealexander** [comment]
All payments are completed automatically, so I've closed the contract. Thanks @gauravahir28!
